vuepress/docs/next/docs/reference/databases.md

```diff
@@ -49,7 +49,7 @@ Entando can also be configured to use an existing
 DBMS provided by the customer. In these situations, lower level database
 operations such as tablespace creation, permissions and clustering must be carried out by the customer. 
 
-Entando then creates and populates the tables, indices and foreign keys. It creates these in the appropriate table 'container' for the DBMS, such
+Entando then creates and populates the tables, indices and foreign keys. It creates them in the appropriate table 'container' for the DBMS, such
```

Comment:
this reads: it creates and populates tables in the appropriate table named container; should remove backticks to make "table" an adj (table container) or remove table altogether for "appropriate `container` - that preserves backticking but doesn't precipitate some weird circular meaning. would condense and simplify so "it creates" doesn't apply to 2 consecutive sentences: "Entando then creates and populates the tables, indices and foreign keys in the appropriate 'container' for the DBMS..."